src/actions/httpErrorAction.js

```diff
@@ -0,0 +1,41 @@
+export const HTTP_401_ERROR = "HTTP_401_ERROR";
```

Comment:
I know exactly from where did you get this code, but friendly reminder: we're using connected-react-router [https://github.com/supasate/connected-react-router/blob/master/FAQ.md#how-to-navigate-with-redux-action](url)